pom.xml

```diff
@@ -7,7 +7,7 @@
     <groupId>io.keyko.monitoring</groupId>
     <artifactId>schemas</artifactId>
     <packaging>jar</packaging>
-    <version>0.2.2</version>
+    <version>0.1.4</version>
```

Comment:
We upgraded to 0.2.0 when we changed the schemas format and we have released v0.2.0, v0.2.1 and v0.2.2. It might be easier to continue with v0.2.3. What do you think @aaitor ?

Reply:
you are right, that version name is wrong. Fixing it!